test/test-parse-var.cpp

```diff
@@ -92,7 +92,10 @@ TEST(test_parse, parse_simple_const) {
   ASSERT_EQ(v.variable_declarations.size(), 1);
```

Comment:
Almost-blocking: Fix the code, not the assertions.
```js
const x = null
```

Make a separate tests specifically for error_missing_initializer_in_const_declaration.